frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss

```diff
@@ -87,9 +101,16 @@
         font-size: 0.75rem;
         letter-spacing: 0.03125rem;
         text-transform: uppercase;
+
+        .posthog-3000 & {
+            background: none;
+        }
+
         > tr {
             > th {
                 font-weight: 700;
+                padding-top: 0.5rem;
+                padding-bottom: 0.5rem;
```

Comment:
Happy for these to not be 3000 only styles?

Reply:
They didn't have any effect in non-3000 because non-3000 has a row "min-height" but good catch, went ahead and moved em.